src/amo/components/Category/index.js

```diff
@@ -24,7 +18,10 @@ export class CategoryBase extends React.Component {
     category: PropTypes.object,
     dispatch: PropTypes.func.isRequired,
     errorHandler: PropTypes.object.isRequired,
+    filters: PropTypes.object,
     loading: PropTypes.boolean,
+    paginationQueryParams: PropTypes.object,
```

Comment:
Yes, it does count. The wrapped component (`Category`) receives the `location` property. To put it another way, when this PropType declaration moves to Flow, the `mapStateToProps` function would trigger an error saying "`ownProps` does not have a `location` property" or something.